modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/throttling/NewThrottleTiersAndLoadOldTiersTestCase.java

```diff
@@ -0,0 +1,610 @@
+                "Response code mismatched when invoke to get Tier Permission Page");
+        assertTrue(applicationPageHttpResponse.getData().contains(TIER_MANAGE_PAGE_APPLICATION_TIER_GOLD),
+                "default tier Gold is not available in Tier Permission page before  add new tear in tiers.xml");
+    }
+
+    private void clean() throws Exception {
+        //remove the Apis
+        apiStoreClientUser.removeApplication(API_TIER_TEST_APP);
+        apiStoreClientUser.removeApplication(APP_TIER_TEST_APP);
+        apiStoreClientUser.removeApplication(RES_TIER_TEST_APP);
+
+        //remove applications
+        deleteAPI(APITierTEST_ID, apiPublisherClientUser);
+        deleteAPI(APPTierTEST_ID, apiPublisherClientUser);
+        deleteAPI(RESTierTEST_ID, apiPublisherClientUser);
+        deleteAPI(Before_TierTEST_ID, apiPublisherClientUser);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void destroy() throws Exception {
+
+        deleteAPI(After_TierTEST_ID, apiPublisherClientUser);
+
+        super.cleanUp();
+    }
+
+    /**
+     * Read the file content and return the content as String.
+     *
+     * @param fileLocation - Location of the file.
+     * @return String - content of the file.
+     * @throws APIManagerIntegrationTestException - exception throws when reading the file.
+     */
+    protected String readFile(String fileLocation) throws APIManagerIntegrationTestException {
+        BufferedReader bufferedReader = null;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(new File(fileLocation)));
+            String line;
+            StringBuilder stringBuilder = new StringBuilder();
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line);
+            }
+            return stringBuilder.toString();
+        } catch (IOException ioE) {
+            throw new APIManagerIntegrationTestException("IOException when reading the file from:" + fileLocation, ioE);
+        } finally {
+            if (bufferedReader != null) {
+                try {
+                    bufferedReader.close();
+                } catch (IOException e) {
+                    log.warn("Error when closing the buffer reade which used to reed the file:" + fileLocation +
+                            ". Error:" + e.getMessage());
+                }
+            }
+        }
+    }
+
+    protected HttpResponse subscribeToAPI(APIIdentifier apiIdentifier, String applicationName,
+            APIStoreRestClient storeRestClient) throws APIManagerIntegrationTestException {
+        SubscriptionRequest subscriptionRequest = new SubscriptionRequest(apiIdentifier.getApiName(),
+                apiIdentifier.getProviderName());
+        subscriptionRequest.setVersion(apiIdentifier.getVersion());
+        subscriptionRequest.setApplicationName(applicationName);
+        if ((apiIdentifier.getTier() != null) && (!apiIdentifier.getTier().equals(""))) {
```

Comment:
already revert changes in commit: https://github.com/wso2/product-apim/commit/a75c4880b08d984d9a0e971387c0d6a07ef2c886